cmd/tealdbg/debugger.go

```diff
@@ -477,9 +535,12 @@ func (d *Debugger) Update(state *logic.DebugState) error {
 
 	// copy state to prevent a data race in this the go-routine and upcoming updates to the state
 	go func(localState logic.DebugState) {
+		// Copy callstack information
+		s.setCallStack(state.CallStack)
```

Comment:
I think `setCallStack` should be within the "updated" notification handing, no need to set it right here since it is not used right away